src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,117 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+
+    private final LocalDate date;
+    private final String time;
+
+    /**
+     * A constructor to initialize a duke.Deadline object.
+     *
+     * @param isDone A boolean to indicate if the deadline is completed.
+     * @param deadlineDescription A string to describe the deadline.
+     * @param index An integer to indicate the position of the duke.Deadline in list of tasks.
+     * @param date A String to indicate the date to be completed by.
+     */
+    public Deadline(boolean isDone, String deadlineDescription, int index, LocalDate date, String time) {
+        super(isDone, deadlineDescription, index);
+        this.date = date;
+        this.time = time;
+    }
+
+    /**
+     * A method to update the deadline from incomplete to complete.
+     */
+    public void markDone() {
+        super.markDone();
+    }
+
+    /**
+     * A method to update the deadline from complete to incomplete.
+     */
+    public void markUndone() {
+        super.markUndone();
+    }
+
+    /**
+     * A method to update the index of the deadline in the list of tasks.
+     *
+     * @param newIndex The most current integer index of the task.
+     */
+    @Override
+    public void setIndex(int newIndex) {
+        super.setIndex(newIndex);
+    }
+
+    /**
+     * Output the deadline added to the list of tasks in the console.
```

Comment:
I think there's a slight grammar mistake here. I noticed the same issue in several other places too.
```suggestion
     * Outputs the deadline added to the list of tasks in the console.
```

Reply:
Changed those to imperative form 👍